storage/README.md

```diff
@@ -28,6 +29,41 @@ amount of data at any time.
 
 ## Samples
 
+### ACL (Access Control Lists)
+
+View the [documentation][acl_docs] or the [source code][acl_code].
+
+__Usage:__ `node acl --help`
+
+```
+Commands:
```

Comment:
I think there's negligible work to be saved by scripting the copy paste. I mean, it could be scripted, but we'd need to devise a `manifest.json` file or something that could give the script meta information about the samples in the directory. Maybe it's not such a bad idea...